tests/dummy/app/templates/ember-flexberry-dummy-application-user-edit.hbs

```diff
@@ -46,7 +46,7 @@
       class=(if model.errors.name "error" "")
       label=(t "forms.ember-flexberry-dummy-application-user-edit.name-caption")
     }}
-    {{flexberry-validationmessage error=(t "forms.ember-flexberry-dummy-application-user-edit.vname-caption") pointing="pointing"}}
+    {{flexberry-validationmessage error=(t "forms.ember-flexberry-dummy-application-user-edit.name-validation-message-caption") pointing="pointing"}}
```

Comment:
Ok. But flexberry-validationmessage should display error object instead of string defined somewhere in translations,
because there could be sereveral validation errors on single model field.
For example model.name could be invalid because:
* it is not defined (if presence: true validator is defined);
* name is too long (if property length validator is defined)
* on so on...

So flexberry-validationmessage in forms templates should look like that:
{{flexberry-validationmessage error=model.errors.name pointing="pointing"}}
instead of
{{flexberry-validationmessage error=(t "forms.ember-flexberry-dummy-application-user-edit.name-validation-message-caption") pointing="pointing"}.